src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -1914,6 +1914,65 @@ class BodyVisitor : public CommonVisitor<BodyVisitor> {
             }
             tmp = ASR::make_ConstantLogical_t(al, x.base.base.loc, result, type);
             return;
+        } else if (call_name == "int") {
+            if (args.size() != 1) {
+                throw SemanticError(call_name + "() takes exactly one argument (" +
+                    std::to_string(args.size()) + " given)", x.base.base.loc);
+            }
```

Comment:
python's `int` also supports base as an argument. See: https://docs.python.org/3/library/functions.html#int

Reply:
Right. I suggest to wait with such arguments once we get the runtime library working, we'll create an `int` function, etc. Then if arguments are known at compile time, we'll evaluate at compile time.